src/compiler/types.ts

```diff
@@ -6010,7 +6010,6 @@ export interface SymbolLinks {
     exportsChecked?: boolean;                   // True if exports of external module have been checked
     typeParametersChecked?: boolean;            // True if type parameters of merged class and interface declarations have been checked.
     isDeclarationWithCollidingName?: boolean;   // True if symbol is block scoped redeclaration
-    bindingElement?: BindingElement;            // Binding element associated with property symbol

```

Comment:
This one has been introduced in https://github.com/microsoft/TypeScript/pull/4598 but its usage was removed in https://github.com/microsoft/TypeScript/pull/59183